python/lsst/cp/pipe/_configure.py

```diff
@@ -15,6 +15,7 @@ def configureIsrTaskLSSTForCalibrations(config):
     config.doCorrectGains = False
     config.doSaturation = False
     config.doSuspect = False
+    config.doITLEdgeBleedMask = False
```

Comment:
This needs to be moved to the new location in application order.  Before `doBias` I guess?